### PR TITLE
Fix radio packets

### PR DESCRIPTION
**Pull request by bbrk24**
Also turn off the LEDs once we receive the start command

### CODE REVIEW

**Review by CClarke513 — APPROVED** [review]
Good work, Riley! I trust the quality of your code

File: frame.h [hunk 1]
```diff
@@ -1,115 +0,0 @@
-#pragma once
```
Comment (DarthSethious1) [inline frame.h:1]: Why was this deleted?

File: sketch_oct9a.ino [hunk 2]
```diff
@@ -198,25 +212,49 @@ void readAltIMU() {
...
+
+  snprintf(
+    buf,
+    sizeof buf,
+    "%c%08X,%c%08X,%07X,%X,%c%02X,%c%02X,%c%02X,%c%04X,%c%04X,%c%04X,%c%04X"
+#if CAPSULE = 2
+    ",%03X,%c%03X,%02X"
+#endif
+    "\n",
+    sign(last_coords.latitude), abs(last_coords.latitude),
```
Comment (DarthSethious1) [inline sketch_oct9a.ino:238]: I understand this isn't a problem, like logic wise it's fine, but why is it done like this? 'sign(thing), abs(thing)'

File: sketch_oct9a.ino [hunk 3]
```diff
@@ -256,7 +294,12 @@ void setup() {
       //The string returned from readStringUntil does not include the terminator
       String command = Serial1.readStringUntil('\n');
 
-      if (command == "wake") {
+      if (command == "start") {
```
Comment (DarthSethious1) [inline sketch_oct9a.ino:297]: still insane that this works

Comment (DarthSethious1) [inline sketch_oct9a.ino:297]: Didn't know where else to put this comment. I understand the logic in 'setup()' that it will never exit this 'do-while' loop, so that's effectively our 'loop' function. I don't understand how this will compile because we don't have a dedicated 'loop' function though. Could you explain that to me?

**Review by DarthSethious1 — COMMENTED** [review]
Overall it looked good, I just don't fully understand the code, so I can't approve yet.

File: frame.h (hunk 1 above)
Comment (DarthSethious1) [reply frame.h:1]: Sorry, why was this 'frame.h' file deleted, not that specific line.

Comment (bbrk24) [reply frame.h:1]: Now that all the packet formatting is done directly in the .ino file, this was no longer necessary.

Comment (DarthSethious1) [reply frame.h:1]: Is it advantageous to do that, or more advantageous to have that frame.h still?

File: sketch_oct9a.ino (hunk 2 above)
Comment (bbrk24) [reply sketch_oct9a.ino:238]: This is a fun thing about C format specifiers.

If I wanted a 3-digit decimal value with `+` or `-` on the beginning, I could just say `%+0.3d` and pass it a signed integer. However, `%+0.3X` doesn't do the same thing for hexadecimal. For whatever reason, C can only print positive hexadecimal values. So instead, I have to pass it `%c%03X`, and give either `'+'` or `'-'` followed by the magnitude of the number.

File: sketch_oct9a.ino (hunk 3 above)
Comment (bbrk24) [reply sketch_oct9a.ino:297]: We do have a dedicated loop function (`void loop()` on line 326, as well as two other loops that run concurrently with it). After receiving the start command, the `break;` statement causes it to exit the while loop in setup.

File: sketch_oct9a.ino (hunk 2 above)
Comment (DarthSethious1) [reply sketch_oct9a.ino:238]: huh, that's funny. How did you figure that out?

File: sketch_oct9a.ino (hunk 3 above)
Comment (DarthSethious1) [reply sketch_oct9a.ino:297]: ahh, gotcha. I didn't see that because that wasn't changed and that was the one section that I didn't manually pull down to look at it.

File: sketch_oct9a.ino (hunk 2 above)
Comment (bbrk24) [reply sketch_oct9a.ino:238]: I tried it: https://tio.run/##S9ZNzknMS///XzkzLzmnNCVVwaa4JCUzXy/DjosrM69EITcxM09DU6GaS0GhoAgokKahpKptoGecEpOnpKOga2hkrmmNLheBLFf7/z8A

File: frame.h (hunk 1 above)
Comment (bbrk24) [reply frame.h:1]: I originally created that file to have formatting (building the `Frame` struct) and actually sending the data be in separate places. Now, the data is formatted into a `char[]` rather than a struct, and I don't believe a C/C++ function can return a plain array like that (instead you have to pass around a pointer to the array).

**Review by DarthSethious1 — APPROVED** [review]
Great job
